index.html

```diff
@@ -530,6 +617,32 @@ <h3>StatusList2021Credential</h3>
   "proof": { ... }
 }
         </pre>
+        <pre class="example nohighlight" title="Example StatusList2021Credential">
+          {
+            "@context": [
+              "https://www.w3.org/2018/credentials/v1",
+              "https://w3id.org/vc/status-list/2021/v1"
+            ],
+            "id": "<span class="highlight">https://example.com/credentials/status/3</span>",
+            "type": ["VerifiableCredential", "<span class="highlight">StatusList2021Credential</span>"],
+            "issuer": "did:example:12345",
+            "issued": "2021-04-05T14:27:40Z",
+            "credentialSubject": {
+              "id": "https://example.com/status/3#list",
+              "type": "<span class="highlight">StatusList2021</span>",
+              "ttl": 500,
+              "statusPurpose": "<span class="highlight">status</span>",
+              "reference": "https://example.org/status-dictionary/"
+              "statusMessages": [ 
```

Comment:
```suggestion
              "size": 2
              "statusMessages": [ 
```
definition of a `size` claim says that if omitted, the value is 1. in the example, `size` is omitted, but more than 2 statuses are being defined, which I found confusing.